restcomm/restcomm.http/src/main/java/org/mobicents/servlet/restcomm/http/AccountsEndpoint.java

```diff
@@ -235,6 +249,23 @@ protected Response putAccount(final MultivaluedMap<String, String> data, final M
                     account = account.setRole(parent.getRole());
                 }
                 dao.addAccount(account);
+
+                // Create default SIP client data
+                MultivaluedMap<String, String> clientData = new MultivaluedMapImpl();
+                clientData.add("Login", data.getFirst("EmailAddress"));
```

Comment:
In case you use the email address for the sip client username, when the user will try to setup a sip client with this username, he will have to url encode the '@' which is bad and we should avoid it.
Instead I suggest we use only the user part of the email address.
For example:
- Account username: administrator@company.com
- Sip client username: administrator